assets/css/app.scss

```diff
@@ -102,6 +105,20 @@ $component-active-bg: $color-eggplant-700;
 
 @import "../node_modules/bootstrap/scss/forms";
 
+[hidden] {
+  display: none !important;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+*:before,
+*:after,
+* {
+  box-sizing: inherit;
+}
+
```

Comment:
I (perosnally) think the box model is easier to reason about with `box-sizing: border-box;` and I think we'd benifit applicaiton wide from using this instead of `content-box` by default. 
You're correct that it does break some things, and I had to patch those to make them continue to work as they used to (vehicle properties panel is an example of this), but I might not have caught everything :/.
I'd be open to moving it into it's own PR/Ticket for 'modernizing' our CSS.

This comes from things like css normalize and css reset type projects([css remedy](https://github.com/jensimmons/cssremedy) being one of my prefered favorites last I checked), which aim to make CSS/HTML styles respond more like people's internal models expect, as well as create a strong baseline for having all browsers render in the same style.

There's a bit of background on this on css-tricks
- https://css-tricks.com/box-sizing/
- https://css-tricks.com/inheriting-box-sizing-probably-slightly-better-best-practice/

Reply:
I agree that `border-box` is an easier mental model to work with most of the time, and the more I learn about CSS the more I wish Skate had started out using one of the available CSS reset libraries. That said, I am a little concerned about how we'll catch all of the possible cases where this might break - VPP is an obvious one, it seems like the sort of thing that will need a lot of QA. I actually wonder if this is a potential use case for automated visual tests.

Reply:
I'm starting to notice more places it broke things (swings view). It's relatively easy to figure out what breaks it by switching between `content-box` and `border-box` repeatedly in dev tools (though time consuming and not an exact science), and I like the idea of having a list of things to fix by having to annotate where we use `content-box` explicitly, but I think I'm gonna localize this style to just the mapSearch page for now, unless y'all can think of a better common denominator.

Reply:
Unfortunately I can't think of a better common denominator, though if you do end up limiting the scope of the change in this PR definitely create a followup tech debt PR to make the change for the whole application.

Reply:
TIL about "CSS resets"! Thanks for the resources on box model - I agree that it seems like an easier mental model to work with.